specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/modelManagement.json

```diff
@@ -0,0 +1,4613 @@
+          },
+          "204": {
+            "description": "The resource does not exist and the request was well formed."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ModelErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Asset"
+        ],
+        "summary": "Update an Asset.",
+        "description": "Patch a specific Asset.",
+        "operationId": "Assets_Patch",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceParameter"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The Id of the Asset to patch.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "patch",
+            "in": "body",
+            "description": "The payload that is used to patch an Asset.",
+            "required": false,
+            "schema": {
+              "uniqueItems": false,
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/JsonPatchOperation"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Asset"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ModelErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/modelmanagement/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.MachineLearningServices/workspaces/{workspace}/images/{id}": {
+      "get": {
+        "tags": [
+          "Image"
+        ],
+        "summary": "Get an Image.",
+        "description": "Get an Image by Id.",
+        "operationId": "Images_QueryById",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceParameter"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The Image Id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "expand",
```

Comment:
`$expand`?

Reply:
Will fix service.

Reply:
Will support both for backward compat and hide 'expand' in Swagger.

Reply:
Fix underway.

Reply:
Postpoing due to back compat issues. Will address in next API version.